NVIDIA/dcgm-exporter
# go version updated to 1.21

Dockerfiles were updated to use GO1.21.

How to test:

1. Build images
```
make local
```
3. Run ubuntu20.04

```
docker run --gpus all --rm -p 9400:9400 docker.io/nvidia/dcgm-exporter:3.3.0-3.2.0-ubuntu20.04
```

4. After ~30 seconds, make an HTTP request:
```
curl -v localhost:9400/metrics
```

You expected to see a list of metrics.

5. Run Dockerfile.ubi8

```
docker run --gpus all --rm -p 9400:9400 docker.io/nvidia/dcgm-exporter:3.3.0-3.2.0-ubi8
```

6. After ~30 seconds, make an HTTP request:
```
curl -v localhost:9400/metrics
```

You expected to see a list of metrics.
